src/components/localstorage/index.js

```diff
@@ -0,0 +1,6 @@
+import LocalStorageReader from './localStorageReader';
+import LocalStorageWritter from './localStorageWritter';
+
+export default {
+	LocalStorageReader, LocalStorageWritter
+};
```

Comment:
this should be a real component (with html), look at the other components/*/index.js